bench/irmin-pack/tree.ml

```diff
@@ -364,39 +371,43 @@ struct
 
   let run_large config =
     reset_stats ();
-    let conf = Irmin_pack.config ~readonly:false ~fresh:true config.root in
-    let* repo = Store.Repo.v conf in
+    let* repo, on_commit, repo_pp = Store.create_repo config.ncommits config in
     let* result =
       Trees.add_large_trees config.width config.nlarge_trees
       |> add_commits ~message:"Playing large mode" repo config.ncommits
+           on_commit
       |> Benchmark.run config
     in
     let+ () = Store.Repo.close repo in
     fun ppf ->
       Format.fprintf ppf
-        "Large trees mode on inode config %a: %d commits, each consisting of \
-         %d large trees of %d entries\n\
+        "Large trees mode on inode config %a, %a: %d commits, each consisting \
+         of %d large trees of %d entries\n\
+         %a@\n\
          %a"
-        pp_inode_config config.inode_config config.ncommits config.nlarge_trees
-        config.width Benchmark.pp_results result
+        pp_inode_config config.inode_config pp_store_type config.store_type
+        config.ncommits config.nlarge_trees config.width repo_pp ()
```

Comment:
you can use `%t` to avoid the extra unit argument for `repo_pp` (as in a previous PR)

Reply:
Thanks, I forgot about this one